packages/material-ui/src/Menu/Menu.js

```diff
@@ -29,92 +28,92 @@ export const styles = {
   },
 };
 
-class Menu extends React.Component {
-  menuListActionsRef = React.createRef();
+const Menu = React.forwardRef(function Menu(props, ref) {
+  const {
+    children,
+    classes,
+    disableAutoFocusItem,
+    MenuListProps,
+    onClose,
+    onEntering,
+    open,
+    PaperProps = {},
+    PopoverClasses,
+    theme,
+    ...other
+  } = props;
+  const menuListActions = React.useRef();
```

Comment:
Ref suffix? @mui-org/core-contributors

Reply:
If I remembered correctly we settled on suffixing return values from `useRef` and `createRef`

Reply:
What about not enforcing any rule for now. We can experiment with different patterns. We will see what work best with the experience. Then settle on something?